adapters/AdColony/AdColonyAdapter/GADMAdapterAdColony.m

```diff
@@ -21,6 +21,9 @@
   INIT_STATE_INITIALIZING
 } InitState;
 
+static NSString *const kAdColonyExplicitConsentGiven = @"explicit_consent_given";
+static NSString *const kAdColonyConsentResponse = @"consent_response";
```

Comment:
Please use the methods mentioned by AdColony in [https://github.com/AdColony/AdColony-iOS-SDK-3/wiki/GDPR](https://github.com/AdColony/AdColony-iOS-SDK-3/wiki/GDPR) to pass the consent value.

Reply:
These are the keys we're using at AdColony for forwarding consent from mediation. They're not publicly documented, but these are indeed the keys we want and that the ad server expects.